tools/crowdpose_concat_train_val.py

```diff
@@ -0,0 +1,49 @@
+# ------------------------------------------------------------------------------
```

Comment:
why add '_concat_' in the file name?

Reply:
This is used to concat train and val data to a single file.